From david-b@pacbell.net Mon Apr 23 10:38:29 2007
From: Li Yang <leoli@freescale.com>
Date: Mon, 23 Apr 2007 10:38:18 -0700
Subject: USB: update MAINAINERS and CREDITS for Freescale USB driver
To: Greg KH <greg@kroah.com>
Cc: Li Yang <leoli@freescale.com>
Message-ID: <200704231038.18792.david-b@pacbell.net>
Content-Disposition: inline

From: Li Yang <leoli@freescale.com>

Add MAINAINERS and CREDITS entry for Freescale Highspeed USB device
driver.

Signed-off-by: Li Yang <leoli@freescale.com>
Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 CREDITS     |    8 ++++++++
 MAINTAINERS |    7 +++++++
 2 files changed, 15 insertions(+)

--- a/CREDITS
+++ b/CREDITS
@@ -3295,6 +3295,14 @@ S: 12725 SW Millikan Way, Suite 400
 S: Beaverton, Oregon 97005
 S: USA
 
+N: Li Yang
+E: leoli@freescale.com
+D: Freescale Highspeed USB device driver
+D: Freescale QE SoC support and Ethernet driver
+S: B-1206 Jingmao Guojigongyu
+S: 16 Baliqiao Nanjie, Beijing 101100
+S: People's Repulic of China
+
 N: Marcelo Tosatti
 E: marcelo@kvack.org
 D: v2.4 kernel maintainer
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -1381,6 +1381,13 @@ L:	linuxppc-embedded@ozlabs.org
 L:	netdev@vger.kernel.org
 S:	Maintained
 
+FREESCALE HIGHSPEED USB DEVICE DRIVER
+P:	Li Yang
+M:	leoli@freescale.com
+L:	linux-usb-devel@lists.sourceforge.net
+L:	linuxppc-embedded@ozlabs.org
+S:	Maintained
+
 FILE LOCKING (flock() and fcntl()/lockf())
 P:	Matthew Wilcox
 M:	matthew@wil.cx
